appium/options/common/common_options.py

```diff
@@ -0,0 +1,644 @@
+    - Set
+        - `None`
+    """
+
+
+class NoResetOption(SupportsCapabilities):
+    NO_RESET = "noReset"
+    no_reset = OptionsDescriptor[Optional[bool], bool](NO_RESET)
+    """
+    Gets and Sets Whether the driver should not perform a reset.
+
+    Usage
+    -----
+    - Get
+        - `self.no_reset`
+    - Set
+        - `self.no_reset` = `value`
+    
+    Parameters
+    ----------
+    `value`: `bool`
+
+    Returns
+    -------
+    - Get
+        - `Optional[bool]`
+    - Set
+        - `None`
+    """
+
+
+class OrientationOption(SupportsCapabilities):
+    ORIENTATION = "orientation"
+    orientation = OptionsDescriptor[Optional[str], str](ORIENTATION)
+    """
+    Gets and Sets the orientation of the device's screen.
+    Usually this is either 'PORTRAIT' or 'LANDSCAPE'.
+
+    Usage
+    -----
+    - Get
+        - `self.orientation`
+    - Set
+        - `self.orientation` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class OtherAppsOption(SupportsCapabilities):
+    OTHER_APPS = "otherApps"
+    other_apps = OptionsDescriptor[Optional[str], str](OTHER_APPS)
+    """
+    Gets and Sets Locations of apps to install before running a test.
+    Each item could be separated with a single comma.
+
+    Usage
+    ----
+    - Get
+        - `self.other_apps`
+    - Set
+        - `self.other_apps` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class PostrunOption(SupportsCapabilities):
+    POSTRUN = "postrun"
+    postrun = OptionsDescriptor[Optional[Dict[str, str]], Dict[str, str]](POSTRUN)
+    """
+    Gets and Sets system script which is supposed to be executed upon
+    driver session quit.
+
+    Usage
+    -----
+    - Get
+        - `self.postrun`
+    - Set
+        - `self.postrun` = `value`
+    
+    Parameters
+    ----------
+    `value`: `Dict[str, str]`
+
+    Returns
+    -------
+    - Get
+        - `Optional[Dict[str, str]]`
+    - Set
+        - `None`
+    """
+
+
+class PrerunOption(SupportsCapabilities):
+    PRERUN = "prerun"
+    prerun = OptionsDescriptor[Optional[Dict[str, str]], Dict[str, str]](PRERUN)
+    """
+    Gets and Sets System script which is supposed to be executed before
+    a driver session is initialised.
+
+    Usage
+    -----
+    - Get
+        - `self.prerun`
+    - Set
+        - `self.prerun` = `value`
+    
+    Parameters
+    ----------
+    `value`: `Dict[str, str]`
+
+    Returns
+    -------
+    - Get
+        - `Optional[Dict[str, str]]`
+    - Set
+        - `None`
+    """
+
+
+class PrintPageSourceOnFindFailureOption(SupportsCapabilities):
+    PRINT_PAGE_SOURCE_ON_FIND_FAILURE = "printPageSourceOnFindFailure"
+    print_page_source_on_find_failure = OptionsDescriptor[Optional[bool], bool]
```

Comment:
this definition must be properly formatted

Reply:
Fixed